engines/gob/detection/tables_urban.h

```diff
@@ -130,13 +130,10 @@
 {
 	{
 		"urban",
-		"Non-Interactive Demo",
-		{
-			{"wdemo.s24", 0, "14ac9bd51db7a075d69ddb144904b271", 87},
-			{"demo.vmd", 0, "65d04715d871c292518b56dd160b0161", 9091237},
-			{"urband.vmd", 0, "60343891868c91854dd5c82766c70ecc", 922461},
-			{0, 0, 0, 0}
-		},
+		"Non-Interactive Demo (v1.00)",
```

Comment:
Why do we need to specify the version? I see there's only one variant

Reply:
The version number was listed in LANGUAGE.INF, so i though i could add it, i remove it aswell if prefered

Reply:
I found the version number in LANUGAGE.INF, so maybe the commit for the version number can be excluded when the PR gets merged. Git/GitHub won't give me the option to amend the commit to remove the version number.